lib/std/heap.zig

```diff
@@ -606,13 +633,13 @@ test "PageAllocator" {
     }
 
     if (builtin.os.tag == .windows) {
-        const slice = try allocator.alignedAlloc(u8, mem.page_size, 128);
+        const slice = try allocator.alignedAlloc(u8, 1, 128);
```

Comment:
Why use `alignedAlloc()` at all here if the alignment is the default alignment of the type?

Reply:
Oversight, I didn't realize that. Will change.

Reply:
Fixed